src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -275,12 +275,14 @@ func _makeBlockRewardMessage(state st.StateTree, minerAddr addr.Address, penalty
 	params[0] = addr.Serialize_Address(minerAddr)
 	params[1] = actor.Serialize_TokenAmount(penalty)
 
+	TODO() // serialize other inputs to BlockRewardMessage or get this from query in RewardActor
+
```

Comment:
The reward actor should query for the miner's power and pledge, not trust the caller. So IMO there should be no more parameters.